rviz_default_plugins/src/rviz_default_plugins/displays/marker/markers/triangle_list_marker.cpp

```diff
@@ -55,6 +62,39 @@ namespace displays
 namespace markers
 {
 
+resource_retriever::MemoryResource getResource(const std::string & resource_path)
+{
+  resource_retriever::Retriever retriever;
+  resource_retriever::MemoryResource res;
+  try {
+    res = retriever.get(resource_path);
+  } catch (resource_retriever::Exception & e) {
+    RVIZ_COMMON_LOG_DEBUG(e.what());
+    return resource_retriever::MemoryResource();
+  }
+
+  return res;
+}
+
+bool loadImage(const Ogre::String & texture_name, const Ogre::String & texture_path)
+{
+  bool image_loaded = false;
+  resource_retriever::MemoryResource res = getResource(texture_path);
+  Ogre::String tex_ext;
```

Comment:
nitpick: I prefer full words for variables names to make it clear what they refer to. E.g. `memory_resource` and `texture_extension`.